### ISSUE TITLE
[Bug] silent crash in release 5.4.0.0 ?

### ISSUE THREAD

**Issue by gilbh** [pit-ray/win-vind#196]
Not sure about this, but I find myself starting vind again from time to time, as if this version of vind, 5.4.0.0, crashes without notice.

Just noting in case you can look into that.

Thanks for all the great work!

**Comment by pit-ray** [comment]
It would be difficult without information I could reproduce.

**Comment by gilbh** [comment]
I am unable to identify conditions for reproducing, other than that it is a portable installation of 5.4.0.0 on the following system:

Edition	Windows 10 Pro
Version	22H2
Installed on	‎3/‎13/‎2021
OS build	19045.3324
Experience	Windows Feature Experience Pack 1000.19041.1000.0

**Comment by Salted-Bread** [comment]
Mine also seems to just silently crash every now and then. Haven't exactly found out what causes it, but it often seems to happen when I play games for a bit. Sometimes it also just seems to quietly exit when I'm in resident mode for some time while watching YouTube. I'll try to keep an eye out to see if I can spot any specific moments/things that make it crash.

I'm using the 64bit version from the installer.

Windows 11 Home
Version 22H2
OS build 22621.2070

Would also be willing to provide log files if there are any.

**Comment by pit-ray** [comment]
@gilbh @Salted-Bread 
Since it is from v5.4.0, I guess it is caused by autocmd. If possible, could you please let me know your autocmd settings?
Thanks.

**Comment by gilbh** [comment]
In my setup the autocmd lines are commented, so I don't have anything to show here.

**Comment by Salted-Bread** [comment]
I don't have anything with autocmd in my config. I don't think it's new to v5.4.0 either. I had the crashes whilst using v.5.2.3 as well.

I tried checking if there are crash logs anywhere that might help. Only thing I found was some dmp files from win-vind in `AppData\Local\CrashDumps`. No idea if they are of any use to you.

**Comment by pit-ray** [comment]
@gilbh  @Salted-Bread 
I'll prepare the same virtual environment as yours and test it. Please wait a moment.

**Comment by tolearnforever** [comment]
hello，i have same problem，please！

**Comment by pit-ray** [comment]
I have not been able to reproduce this yet. In previous versions, I have had silent crashes, but not at all recently.

It may be an environment-specific issue, such as blocking by security programs. I will continue to investigate.
Please give me some information if you find out anything.

**Comment by romek-dev** [comment]
Hi, i've recently started using win-vind on two of my machines running windows 11. It keeps silently crashing as well on both of the machines and i have to restart it. Is there any log file i could look into and maybe attach to this issue?

**Comment by pit-ray** [comment]
@romek-dev 
Sorry for my late reply.

The second error says the values of some parameters are invalid (cannot be understood as numbers) since the stoi function is only used in the parameter parsing in win-vind.

So, if your .vindrc is customized, please check the syntax or value types (specifically, the `gridmove_size` option or `sleep` command, and so on).

**Comment by Cattle0Horse** [comment]
I also encountered the same problem
The following is the log output
```
========== System Infomation ==========
[Windows]
      Edition: Windows 10 Pro
      Version: 22H2
Build Numbers: 10.0.19045
 Architecture: x64

[win-vind]
      Version: 5.11.3.0
=======================================
[Message] Version: tiny
[Error] No mapping for the Unicode character exists in the target multi-byte code page.
[Error] 在多字节的目标代码页中，没有此 Unicode 字符可以映射到的字符。
```

My configuration file is as follows
```
version tiny
set initmode=i  " Insert mode
set cmd_fadeout = 0
imap <esc-left> <f20>
imap <ctrl-]> <f20>
imap <f8> <f20>
imap <esc-down> <f20>
imap <rshift-space> <easyclick><click_left>
set uiacachebuild
set uiacachebuild_lifetime=5000
set uiacachebuild_staybegin=500
set uiacachebuild_stayend=2000
```

**Comment by pit-ray** [comment]
Locking the computer may be a problem as mentioned in #252.

**Comment by pit-ray** [comment]
Probably this issue has been resolved in v5.13.1.
Please report any problems again.
